nixos/modules/services/continuous-integration/hydra/default.nix

```
@@ -280,9 +280,12 @@ in
       keep-outputs = true
       keep-derivations = true
 
-      # The default (`true') slows Nix down a lot since the build farm
-      # has so many GC roots.
-      gc-check-reachability = false
+
+      ${optionalString (versionOlder (getVersion config.nix.package.out) "2.4pre") ''
+        # The default (`true') slows Nix down a lot since the build farm
+        # has so many GC roots.
+        gc-check-reachability = false
+      ''}
     '';
```

```suggestion
    '' + optionalString (versionOlder (getVersion config.nix.package.out) "2.4pre") ''
      # The default (`true') slows Nix down a lot since the build farm
      # has so many GC roots.
      gc-check-reachability = false
    '';
```
avoids the string var nesting.